mlpf/tfmodel/utils.py

```diff
@@ -565,11 +550,25 @@ def gen_jet_loss(y_true, y_pred):
         px = y[typ][..., 0] * y[typ][..., 4]
         py = y[typ][..., 0] * y[typ][..., 3]
         jet_pt[typ] = batched_jet_reco(px, py, jet_idx, max_jets)
+    return jet_pt
```

Comment:
perhaps rename this function to `compute_jet_pt` or something like that since it's not a loss anymore?